tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -63,6 +63,18 @@ export class Ide {
         }
     }
 
+    async verifyAndSwitchToIdeFrame(timeout: number = TimeoutConstants.TS_IDE_LOAD_TIMEOUT) {
```

Comment:
@makambalaji  This should be handled in code in `testsLibrary/ProjectAndFileTests.ts`, there's no point in adding this extra method.

Reply:
In multiple files, using the same logic, So IMO, its good to keep this function

Reply:
I can only see the method called at one place in your PR.

Reply:
tests/e2e/pageobjects/ide/ProjectTree.ts, I used twice, For other spec file fixes. we can reuse this function